src/media/css/app-list.styl

```diff
@@ -117,9 +117,19 @@ section.langpacks > div {
         &:nth-child(2n) {
             border-right: 0;
             padding-right: 0;
+
+            +rtl() {
+                border-right: $border;
+                padding: 20px 50px 20px 0;
```

Comment:
Can we set `.app-list-app { padding: 20px 50px; }; and then just use `padding-left: 0;` here? Might need another rule for the first (or last) child. I can't tell exactly.